### PR TITLE
add docs site for 0.10 beta release

### PR DESCRIPTION
**Pull request by Bradamant3**
Signed-off-by: JENNIFER RONDEAU <jrondeau@heptio.com>

For review: https://bradamant3.github.io/ark/v0.10.0-beta.1/

n.b. breaking changes warning in `index.html` is manually added; will add PR for master branch
please double-check TOC completeness

### CODE REVIEW

**Comment by skriss** [comment]
Still poking around but this LGTM - @wwitzel3 @nrb @carlisia PTAL and see if all looks good (I'm viewing in Jennifer's fork per above link)

**Comment by carlisia** [comment]
Looking..

File: _config.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name: ark
 baseurl: /ark
 google_analytics: FILL_OUT
-latest: v0.9.0
```
Comment (nrb) [inline _config.yml]: Do we want latest to point to this still? Or only let people see it if they're specifically looking for it?

Comment (ncdc) [reply _config.yml]: We should continue to show 0.9 as the latest stable imho

Comment (Bradamant3) [reply _config.yml]: Only sorta done. It's taking me a little while to get the conditional logic right for a better version warning statement for 0.10.0-beta so I figure it's better to get at least a partial fix up first. Will continue to noodle (conditional logic brain evidently sleepy atm, also lazy)

Note that version warning logic doesn't work, but doesn't break.

**Comment by Bradamant3** [comment]
See my comment in latest commit

File: _includes/version-warning.html [hunk 2]
```diff
@@ -7,6 +7,15 @@
     <a href="{{latest_url}}">latest version</a>.
     </p>
   </div>
+{% elsif page.version == 'v.0.10.0-beta.1' %}
```
Comment (skriss) [inline _includes/version-warning.html]: looks like you've got an extraneous `.` after `v`

Comment (Bradamant3) [reply _includes/version-warning.html]: lolyep of course you know when I saw it ...

**Pull request merged**